lib/gtfs_server/agencies/trimet/service.ex

```diff
@@ -0,0 +1,67 @@
+defmodule GtfsServer.Agencies.Trimet.Service do
+
+  import Utility, only: [
+    create_vehicle_position_key: 1,
```

Comment:
Care with imports, generally not preferred because context removal, DSLs make more sense, example would be writing a utility module for Conn because it's standard to import the entire module for Plug.Conn already, if you had a function called render_json or json_error I could see the validity of an import. But outside DSL and standards preferable to to use it. Once you've hopping in some big projects with hundreds of files this will make a lot more sense